dictmaker.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * This script generates a new words_dictionary.json from words_alpha.txt
+ * 
+ * You must have nodejs installed to run this script.
+ * 
+ * To use it, run this from the command line in the project folder:
+ * node dictmaker.js
+ * 
+ */
+
+var FS = require('fs');
+
+var SOURCE_PATH = './words_alpha.txt';
+var OUTPUT_PATH = './words_dictionary.json';
+
+
+console.log('reading file ' + SOURCE_PATH);
+
+FS.readFile(SOURCE_PATH, 'utf8', function(err, content) {
+	if(err) {
+		throw err;
+	}
+	
+	console.log('processing word list...');
+	
+	var words = content.split('\n');
```

Comment:
`var words` does not _appear_ to be used below ... 🤔

Reply:
Doh! That'll teach me to code on Xmas.